tutorials/misc/saving_games.rst

```diff
@@ -95,25 +135,46 @@ way to pull the data out of the file as well.
         save_game.open("user://savegame.save", File.WRITE)
         var save_nodes = get_tree().get_nodes_in_group("Persist")
         for i in save_nodes:
-            var node_data = i.save()
+            var node_data = i.call("save");
             save_game.store_line(to_json(node_data))
         save_game.close()
 
+ .. code-tab:: csharp
+
+    // Note: This can be called from anywhere inside the tree.  This function is path independent.
+    // Go through everything in the persist category and ask them to return a dict of relevant variables
+    public void SaveGame()
+    {
+        var saveGame = new File();
+        saveGame.Open("user://savegame.save", (int)File.ModeFlags.Write);
+
+        var saveNodes = GetTree().GetNodesInGroup("Persist");
```

Comment:
For some reason, I believe `SceneTree.GetNodesInGroup` returns `object[]` now. Probably we should report an issue about it, but for now we'd need a casting to make this compile.

Reply:
This already compiles because `foreach` is implicitely casting from `object` to `Node` here.  
Do you think the docs would be more readable with an explicit cast? I wondered that when I was writing it but ended up thinking it was kind of useless.

Reply:
Oh, I didn't know `foreach` would implictly cast the items it iterates. It seems that my limited experience of the language shows here :sweat_smile: Sorry about that!

The PR looks good to me.